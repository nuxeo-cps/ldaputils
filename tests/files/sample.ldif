# extended LDIF
#
# LDAPv3
# base <ou=ministere,o=gouv,c=fr> with scope subtree
# filter: (objectclass=*)
# requesting: ALL
#

# MINISTERE, GOUV, FR
dn: ou=MINISTERE,o=GOUV,c=FR
objectClass: organizationalUnit
ou: MINISTERE

# joeuser, MINISTERE, GOUV, FR
dn: uid=joeuser,ou=MINISTERE,o=GOUV,c=FR
objectClass: top
objectClass: person
objectClass: inetOrgPerson
uid: joeuser
sn: Jack
givenName: Jojo
cn: FEAR
mail: joeuser@example.net

# germaine.duchemin, MINISTERE, GOUV, FR
dn: uid=germaine.duchemin,ou=MINISTERE,o=GOUV,c=FR
objectClass: top
objectClass: person
objectClass: organizationalPerson
objectClass: inetOrgPerson
cn: DUCHEMIN Germaine
displayName: DUCHEMIN Germaine
sn: DUCHEMIN
mail: Germaine.Duchemin@ministere.fr
businessCategory: Bureau
ou: MINISTERE
uid: germaine.duchemin
userPassword:: Q0hBTkdFTUU=
title: Madame
employeeNumber: 1
givenName: Germaine
telephoneNumber: 01.53.94.88.20
description: Affaires

