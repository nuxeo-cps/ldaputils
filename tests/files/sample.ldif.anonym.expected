dn: ou=MINISTERE,o=GOUV,c=FR
objectClass: organizationalUnit
ou: MINISTERE

dn: uid=0,ou=MINISTERE,o=GOUV,c=FR
cn: 1
givenName: 4
mail: a3@example.net
objectClass: top
objectClass: person
objectClass: inetOrgPerson
sn: 2
uid: 0

dn: uid=5,ou=MINISTERE,o=GOUV,c=FR
businessCategory: 9
cn: 8
description: 12
displayName: 7
employeeNumber: 13
givenName: 16
mail: a15@example.net
objectClass: top
objectClass: person
objectClass: organizationalPerson
objectClass: inetOrgPerson
ou: MINISTERE
sn: 14
telephoneNumber: 6
title: 10
uid: 5
userPassword: 11

